package.json

```diff
@@ -16,6 +16,7 @@
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
     "@fontsource/nunito": "^4.5.9",
+    "@reduxjs/toolkit": "^1.8.2",
```

Comment:
@brixtuyor please exclude this for now since it's not included in this task scope and this package is not being used at the moment.